tensorbay/label/label_polygon.py

```diff
@@ -86,6 +86,69 @@ def __init__(self, is_tracking: bool = False) -> None:
         IsTrackingMixin.__init__(self, is_tracking)
 
 
+@SubcatalogTypeRegister(LabelType.MULTI_POLYGON)
+class MultiPolygonSubcatalog(  # pylint: disable=too-many-ancestors
+    SubcatalogBase, IsTrackingMixin, CategoriesMixin, AttributesMixin
+):
+    """This class defines the subcatalog for multiple polygon type of labels.
+
+    Arguments:
+        is_tracking: A boolean value indicates whether the corresponding
+            subcatalog contains tracking information.
+
+    Attributes:
+        description: The description of the entire multiple polygon subcatalog.
+        categories: All the possible categories in the corresponding dataset
+            stored in a :class:`~tensorbay.utility.name.NameList`
+            with the category names as keys
+            and the :class:`~tensorbay.label.supports.CategoryInfo` as values.
+        category_delimiter: The delimiter in category values indicating parent-child relationship.
+        attributes: All the possible attributes in the corresponding dataset
+            stored in a :class:`~tensorbay.utility.name.NameList`
+            with the attribute names as keys
+            and the :class:`~tensorbay.label.attribute.AttributeInfo` as values.
+        is_tracking: Whether the Subcatalog contains tracking information.
+
+    Examples:
+        *Initialization Method 1:* Init from ``MultiPolygonSubcatalog.loads()`` method.
+
+        >>> catalog = {
+        ...     "MULTI_POLYGON": {
+        ...         "isTracking": True,
+        ...         "categories": [{"name": "0"}, {"name": "1"}],
+        ...         "attributes": [{"name": "gender", "enum": ["male", "female"]}],
+        ...     }
+        ... }
+        >>> MultiPolygonSubcatalog.loads(catalog["MULTI_POLYGON"])
+        MultiPolygonSubcatalog(
+          (is_tracking): True,
+          (categories): NameList [...],
+          (attributes): NameList [...]
+        )
+
+        *Initialization Method 2:* Init an empty MultiPolygonSubcatalog
+        and then add the attributes.
+
+        >>> from tensorbay.label import CategoryInfo, AttributeInfo
+        >>> multi_polygon_subcatalog = MultiPolygonSubcatalog()
+        >>> multi_polygon_subcatalog.is_tracking = True
+        >>> multi_polygon_subcatalog.add_category(CategoryInfo("a"))
+        >>> multi_polygon_subcatalog.add_attribute(
+            AttributeInfo("gender", enum=["female", "male"]))
```

Comment:
No need to use `CategoryInfo` and `AttributeInfo`.

Reply:
Changed into str